test/patterns/cltools/fileCompile/chplCompile.chpl

```diff
@@ -0,0 +1,54 @@
+
+/*
+A tool written in chapel to compile a chapel program and to demonstrate the use of the 
+spawn module and how it is used to create subprocesses within a chapel program. 
+*/
+module chplCompile {
+
+use Spawn;
+
+//Command to be executed. Starts with chpl to signal the chapel compiler.
+var command: string = "chpl";
```

Comment:
A top level `var` is okay, but it might be better to pass around a command `var` with `"chpl "` as a `const` to prepend to the command.